schema/pod.go

```diff
@@ -100,13 +135,13 @@ type RuntimeApp struct {
 	Name        types.ACName      `json:"name"`
 	Image       RuntimeImage      `json:"image"`
 	App         *types.App        `json:"app,omitempty"`
-	Mounts      []Mount           `json:"mounts"`
-	Annotations types.Annotations `json:"annotations"`
+	Mounts      []Mount           `json:"mounts,omitempty"`
```

Comment:
We need to go through and audit all these fields to ensure that "optional" and "required" (from the spec) are implemented appropriately here in the schema code.